lib/SearchAndSort/SearchAndSort.js

```diff
@@ -170,6 +170,7 @@ class SearchAndSort extends React.Component {
       connect: PropTypes.func,
       hasPerm: PropTypes.func.isRequired
     }).isRequired,
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
Could it be just `PropTypes.node` ?

Reply:
`Pane` accepts `PropTypes.oneOfType([PropTypes.string, PropTypes.element])`